server/src/main/java/org/apache/gravitino/server/web/rest/ModelOperations.java

```diff
@@ -662,6 +673,86 @@ public Response alterModel(
     }
   }
 
+  @GET
+  @Path("{model}/versions/{version}/uri")
```

Comment:
I would suggest to use path parameter, rather than query parameter for uri name, like `versions/{version}/uris/{uriName}`.